test-d/formio/components/radio.test-d.ts

```diff
@@ -0,0 +1,200 @@
+import {expectAssignable, expectNotAssignable} from 'tsd';
+
+import {RadioComponentSchema} from '../../../lib';
+
+// minimal component schema, manual:
+expectAssignable<RadioComponentSchema>({
+  id: 'yejak',
+  type: 'radio',
+  key: 'aRadio',
+  label: 'A radio',
+  openForms: {
+    dataSrc: 'manual',
+    translations: {},
+  },
+  values: [
+    {
+      value: 'dummy',
+      label: 'dummy',
+    }
+  ],
+});
+
+// minimal component schema, variable:
+expectAssignable<RadioComponentSchema>({
+  id: 'yejak',
+  type: 'radio',
+  key: 'aRadio',
+  label: 'A radio',
+  openForms: {
+    dataSrc: 'variable',
+    itemsExpression: 'dummy',
```

Comment:
```suggestion
    itemsExpression: {var: 'dummy'},
```
a bit more representative :)